0019-third-party-manufacturers.md

```diff
@@ -79,7 +85,14 @@ Lastly, manufacturers are expected to provide:
 While we recognize that much of this information could be considered sensitive or proprietary, we believe it is imperative to build trust with the initial set of manufacturers that are brought in through this process. Thus we ask more of this initial set of manufacturers than we might from later approvals.
 
 
+### Light Hotspot Requirements (Amendment)
+With the incoming launch of Validators (HIP25), the network no longer requires compute-intensive Hotspots to run consensus groups. This reduces the requirements for Hotspots such that they become "Light Hotspots". Light Hotspots are a new subclass of trusted Hotspots on the Network, able to participate in PoC, Witness, and create Challenges. In addition to PoC activities, they can transfer Data Packets. Light Hotspots will earn HNT for these activities. 
+
+Light Hotspots will **not** participate in Consensus and will be flagged as such on the Blockchain. They will not mine HNT for participating in Consensus.
+
+The reduced compute requirements will enable a new class of lower-cost Hotspots to enter the market.
 
+Light Hotspots will not be expected to pass the Audit and Compute requirements to run and participate in consensus.
```

Comment:
```suggestion
Light Hotspots will not be expected to pass the Audit and Compute requirements to run and participate in consensus. Their block absorption rate may be documented as a comparison to other blockchain following miners. 
```

Reply:
Maybe worthwhile addition. We want to make sure Light Hotspots can keep up with the chain so they can perform PoC activities normally.